File: src/main/java/com/meetify/server/repository/PlaceRepository.java
package com.meetify.server.repository;

import com.meetify.server.model.Id;
import com.meetify.server.model.Place;
import org.springframework.data.jpa.repository.JpaRepository;

import java.util.Optional;

/**
 * com.meetify.server.repository
 * Created by kr3v on 04.10.2016.
 */

public interface PlaceRepository extends JpaRepository<Place, Long> {
    Optional<Place> findById(Id id);
}

File: src/main/java/com/meetify/server/repository/PlaceRepository.kt
package com.meetify.server.repository

import com.meetify.server.model.Place

/**
 * Interface, that used to provide layer between database and server logic by autowiring by server.
 * This interface represents places.
 * @author      Dmitry Baynak
 * @version     0.0.1
 * @since       0.0.1
 */
interface PlaceRepository : BaseRepository<Place>
